### DISCUSSION TITLE
افزودن کانفیگ ساده ی vless tcp

### DISCUSSION THREAD

**Discussion by metalgold23** [hiddify/Hiddify-Manager#4036] [Ideas ایدهها]
سلام
لطفا اگر امکان دارد این سه مورد رو بررسی کنید.
1. کانفیگ ساده ی vless tcp بدون L3 رو به پنل اضافه کنید.
vless
tcp
head type:none

جهت استفاده ی این کانفیگ برای ipv6 و همچنین تانل ساده.

2. بارکد های کانفیگ ها رو مثل قبل ساده کنید.گوشی های با دوربین ضعیف مشکل دارن.
3.بارکد صفحه ی اصلی را لینک سابسکریپشن یا خودکار بزارید،موقع خواندن بعضی برنامه ها مثل ویتوری.کانفیگ های ریلیتی کار نمیکنن.
ممنون از تیم قوی هیدیفای

**Comment by Ataheritar** [comment]
لطفا مدیران محترم هیدیفای انجامش بدن. من بیش از یک سال هست هیدیفای دارم و واقعا لذت بردم ازش.
ولی الان با شرایط جدید که نیاز ضروری به تانل هست و ipv6 ، بخاطر نبود همین کانفیگ، علی رغم میل باطنی، ناچار شدم به xui رو بیارم.
واقعا متوجه علت مقاومت شما نمیشم.

**Comment by askih0055** [comment]
سلام مدیران عزیز هیدیفای. شاید این متد قبلا نیاز نبود  ولی الان نیاز شدید به این پروتکول  هست 
بخاطر شرایط  به وجود آمدن همچین شرایط پیچیده که نیاز داریم به تانل ipv6  ، که شدیدا به این پروتکول نیاز داریم 
 تقاضا میشود که این متد را در پنل هیدیفای اضافه کنید ممون

**Comment by mrbl69** [comment]
هیدیفای،انجامش بده لعنتی،انجام بده.

**Comment by zZedix** [comment]
کانفیگ VLESS بدون TLS هیچ رمزنگاری نداره و اصلا امنیت نداره
اگربه امنیت کاربرانتون اهمیت میدید از این کانفیگ فاصله بگیرید
اگر میخواید از این کانفیگ استفاده کنید باید بجاش از کانفیگ VMESS TCP استفاده کنید که خودش رمزنگاری داره و دارای امنیت
مجدد تکرار میکنم کانفیگ VLESS رو بدون TLS هرگز استفاده نکنید
به منیجر هم اضافه نخواهد شد چون استاندارد نیست

**Reply by metalgold23** [reply]
سلام مهندس.
لااقل کانفیگ کلود که پچ طولانی میشه گذاشت رو اوکی کنید! این که امنیتش بیشتره. شما رو نمیدونیم ولی ما که حرص میخوریم یه پنل ساده همچی کاری کنه و هیدیفای نه.
محدودیت آپلود دانلود کلودفر رو دور میزنه.
دامنه
cloudmehranmmdv2ray.v2rayng6.6.6.6.ir.Com.Site.xn--fhq0la447aac5504dea701axr1g.example.com
پچ
/t.me/V2rayFa-V2rayng-v2rayNgfree-telegram-t.me/V2rayfa_telegram?ed=2048
